conf/messages.en

```diff
@@ -16,7 +16,7 @@ site.start = Start
 
 site.continue = Continue
 site.saveAndContinue = Save and continue
-site.agreeAndContinue = Agree and continue
+site.agreeAndContinue = Agree and submit
```

Comment:
Do you need to do the same change [here](https://github.com/hmrc/pension-scheme-return-frontend/blob/5f69fc5d2e3af62213d8dee1744ddc54a38ed541/conf/messages.en#L339) and [here](https://github.com/hmrc/pension-scheme-return-frontend/blob/5f69fc5d2e3af62213d8dee1744ddc54a38ed541/conf/messages.en#L355) too?